src/Components/Header/Header.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+import App from "../../Containers/App/App";
+import { connect } from "react-redux";
+import { fetchAllNotes } from "../../Thunks/fetchAllNotes";
+import "./Header.scss";
+
+export const Header = class extends Component {
```

Comment:
I've never seen this syntax before.  Do you think that we'll need the header to be a class?